apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -26,16 +26,24 @@ namespace Google.Cloud.Spanner.Data
 {
     public sealed partial class SpannerDbType
     {
-        internal object ConvertToClrType(Value protobufValue) => ConvertToClrType(
-            protobufValue, typeof(object));
+        internal object ConvertToClrType(Value protobufValue, SpannerConversionOptions options) =>
+            ConvertToClrType(protobufValue, typeof(object), options, true);
```

Comment:
Nit: perhaps use a named argument for this bool literal here so it is a bit more self-documenting. Same comment on the next one.

Reply:
Done.